### PR TITLE
Add prod entry for concise and chain

### PR DESCRIPTION
**Pull request by frankinspace**

### CODE REVIEW

**Review by chris-durbin — APPROVED** [review]
I tested out by pointing my local environment at production. I had to make one change due to a harmony bug to just add an empty collections list to the `podaac/concise` configuration.

File: config/services.yml [hunk 1]
```diff
@@ -258,6 +258,90 @@ https://cmr.earthdata.nasa.gov:
...
+      <<: *default-argo-config
+      params:
+        <<: *default-argo-params
+        image: !Env ${PODAAC_CONCISE_IMAGE}
+        image_pull_policy: !Env ${PODAAC_CONCISE_IMAGE_PULL_POLICY}
+        template_type: chaining
+        env:
+          <<: *default-argo-env
+          STAGING_PATH: public/podaac/l2-subsetter-concise
+    collections:
```
Comment (chris-durbin) [inline config/services.yml:300]: Seems like moving this to use `umm_s` as well is a good idea in the future.

**Review by vinnyinverso — APPROVED** [review]

File: config/services.yml (hunk 1 above)
Comment (frankinspace) [reply config/services.yml:300]: I wasn't sure if that was possible with a chain?

Comment (chris-durbin) [reply config/services.yml:300]: I thought there was an idea that each chain would be its own UMM-S record, but not sure where that ended up in practice. @bilts or @asteiker would know best.

**Pull request merged**